ag_data/presets/helpers.py

```diff
@@ -0,0 +1,104 @@
+from ag_data import models
+from ag_data.presets import sample_user_data
+from ag_data import utilities
+
+
+def createVenueFromPresetAtIndex(index):
+    """Create a venue from available presets of venues
+
+    Arguments:
+
+        index {int} -- the index of the sensor preset to use.
+
+    Raises:
+
+        Exception: an exception raises when the index is not valid in presets.
+    """
+
+    if index > len(sample_user_data.sample_venues) - 1:
+        raise Exception(
+            "Cannot find requested venue (index " + str(index) + ") from presets"
```

Comment:
Nit: Use f-string.

```
f"Cannot find requested venue (index {index}) from presets"
```

Reply:
Appreciate this genius tip! Will change!

Reply:
This is now in #376.